### DISCUSSION TITLE
UEFI Network Boot:  exec format error

### DISCUSSION THREAD

**Discussion by jonelinux** [ipxe/ipxe#1108] [General]
I want to integrate MaxDOS  into ipxe-uefi， but not working ,how can i do?
:maxdos
  initrd ${xieyi}://${next-server}/ipxe/maxdos.img
  kernel ${xieyi}://${next-server}/ipxe/memdisk initrd=maxdos.img
  boot
------------------------------------------------
![75ed5e5bc47422176f205aa2a64d722](https://github.com/ipxe/ipxe/assets/31331818/c8225d02-ac51-4330-9e61-bbd7d289b70b)

**Comment by NiKiZe** [comment]
Start with making it run I efi at all (not possible with DOS) than you can use a number of ways to boot that EFI application from iPXE.
